ROMUtils.cpp

```diff
@@ -55,11 +55,75 @@ namespace ROMUtils
     /// A pointer into the ROM data to start reading from.
     /// </param>
     /// <param name="outputSize">
-    /// The predicted size of the output data.
+    /// The predicted size of the output data.(unit: Byte)
     /// </param>
     /// <return>A pointer to decompressed data.</return>
-    unsigned char *RLEDecompress(unsigned char *data, int outputSize)
+    unsigned char *RLEDecompress(unsigned char *data, int address, int outputSize)
```

Comment:
We don't need "address" if we make data point to the compressed data

Reply:
there are several choices we can do:
1 unsigned char *data directly point to the real data.
2. use 2 parameters like the upper ones
3. only input the virtual address in ROM and operate all the things internally.